README.md

```diff
@@ -1,7 +1,33 @@
-## Erpnext Autoinstall
 
-descr
+[uri_license]: http://www.gnu.org/licenses/agpl.html
+[uri_license_image]: https://img.shields.io/badge/License-AGPL%20v3-blue.svg
+[![Build Status](https://travis-ci.org/Monogramm/erpnext_autoinstall.svg)](https://travis-ci.org/Monogramm/erpnext_autoinstall)
+
+[![License: AGPL v3][uri_license_image]][uri_license]
+
+## ERPNext AutoInstall
+
+:alembic: **Experimental** ERPNext application for automatic setup.
+
+The objective is to setup ERPNext automatically based on environment variables, mainly for docker usage.
 
 #### License
 
-AGPL
\ No newline at end of file
+AGPL
+
+## Installation
+
+  ```
+  bench get-app --branch develop erpnext_autoinstall https://github.com/Monogramm/erpnext_autoinstall
+  bench install-app erpnext_autoinstall
+  ```
+
+## How to use
+
+Environment variables for automatic setup:
+* WebSite configuration
+  * **TODO**
+* Email configuration
+  * **TODO**
+* LDAP configuration
+  * **TODO**
```

Comment:
@AminovE99 Need to list the available environment variables.

Reply:
Done, I think

Reply:
@AminovE99 nope, you added the environment variables in `.env` file, but not in the `README.md` documentation.